src/main/java/com/utilaider/logistics/utility/SecurityConfig.java

```diff
@@ -26,9 +26,9 @@ public void configure(WebSecurity web) throws Exception {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        http.addFilterBefore(new SimpleCORSFilter(), ChannelProcessingFilter.class)
+        http.csrf().disable().addFilterBefore(new SimpleCORSFilter(), ChannelProcessingFilter.class)
                 .authorizeRequests()
-                .antMatchers("/login/**", "/registration/**", "/logout/**","/test").permitAll()
+                .antMatchers("/login/**", "/registration/**", "/logout/**","/reg/**").permitAll()

```

Comment:
In above comment I already said this is used testing of registration jsp. without permit I can't access that's why I mention reg in ant matchers.